janusgraph-dist/src/assembly/static/bin/janusgraph-server.sh

```diff
@@ -233,28 +243,32 @@ startForeground() {
   fi
 
   if [[ -z "$RUNAS" ]]; then
+    echo "$JANUSGRAPH_YAML will be used to start JanusGraph Server in foreground"
     exec $JAVA -Dlog4j.configuration=$LOG4J_CONF $JAVA_OPTIONS -cp $CLASSPATH $JANUSGRAPH_SERVER_CMD "$JANUSGRAPH_YAML"
     exit 0
   else
     echo Starting in foreground not supported with RUNAS
     exit 1
   fi
-
 }
+
 printUsage() {
-  echo "Usage: $0 {start|stop|restart|status|console|usage <group> <artifact> <version>|<conf file>}"
+  echo "Usage: $0 {start [conf file]|stop|restart [conf file]|status|console|usage <group> <artifact> <version>|<conf file>}"
   echo
-  echo "    start           Start the server in the background using conf/gremlin-server/gremlin-server.yaml as the"
-  echo "                    default configuration file"
+  echo "    start           Start the server in the background. Configuration file can be specified as a second argument
+                    or as JANUSGRAPH_YAML environment variable. If configuration file is not specified
+                    or has invalid path than JanusGraph server will try to use the default configuration file
+                    at relative location conf/gremlin-server/gremlin-server.yaml"
   echo "    stop            Stop the server"
-  echo "    restart         Stop and start the server"
+  echo "    restart         Stop and start the server. To use previously used configuration it should be specified again
+                    as described in \"start\" command"
   echo "    status          Check if the server is running"
-  echo "    console         Start the server in the foreground using conf/gremlin-server/gremlin-server.yaml as the"
-  echo "                    default configuration file"
+  echo "    console         Start the server in the foreground. Same rules are applied for configurations as described
+                    in \"start\" command"
   echo "    usage           Print out this help message"
   echo
-  echo "If using a custom YAML configuration file then specify it as the only argument for a JanusGraph"
-  echo "Server to run in the foreground or specify it via the JANUSGRAPH_YAML environment variable."
+  echo "In case command is not specified and the configuration is specified as a second argument, JanusGraph Server will
```

Comment:
(nitpick) Isn't the configuration the first and only argument here and not the second argument?

Reply:
You are right it should be `the first` here. I fixed that comment and also removed a duplication block.
The diff is here: https://github.com/JanusGraph/janusgraph/compare/08e3fdaf36cbed0ca6c74ccd3d52ef43b5801896..eac1058966c40947f20b2f313ceec45a0d641692